kamel run --dev examples/resources-route.groovy issues Cannot find resource

kamel run --dev examples/resources-route.groovy issues the following log:

1] 2019-05-28 13:55:02.439 ERROR [Camel (camel-k) thread #2 - timer://resources] DefaultErrorHandler - Failed delivery for (MessageId: ID-resources-route-cd6ccf485-5zsm8-1559051699916-0-2 on ExchangeId: ID-resources-route-cd6ccf485-5zsm8-1559051699916-0-1). Exhausted after delivery attempt: 1 caught: org.apache.camel.CamelExecutionException: Exception occurred during execution on the exchange: Exchange[ID-resources-route-cd6ccf485-5zsm8-1559051699916-0-1]
[1] Message History
[1] ---------------------------------------------------------------------------------------------------------------------------------------
[1] RouteId              ProcessorId          Processor                                                                        Elapsed (ms)
[1] [resources         ] [resources         ] [timer://resources                                                             ] [        10]
[1] [resources         ] [setBody1          ] [setBody[simple{resource:platform:resources-data.txt}]                         ] [         6]
[1] Stacktrace
[1] ---------------------------------------------------------------------------------------------------------------------------------------
[1] org.apache.camel.CamelExecutionException: Exception occurred during execution on the exchange: Exchange[ID-resources-route-cd6ccf485-5zsm8-1559051699916-0-1]
[1] 	at org.apache.camel.util.ObjectHelper.wrapCamelExecutionException(ObjectHelper.java:1842) ~[org.apache.camel.camel-core-2.23.2.jar:2.23.2]
[1] 	at org.apache.camel.builder.SimpleBuilder.createExpression(SimpleBuilder.java:117) ~[org.apache.camel.camel-core-2.23.2.jar:2.23.2]
[1] 	at org.apache.camel.builder.SimpleBuilder.evaluate(SimpleBuilder.java:90) ~[org.apache.camel.camel-core-2.23.2.jar:2.23.2]
[1] 	at org.apache.camel.processor.SetBodyProcessor.process(SetBodyProcessor.java:49) ~[org.apache.camel.camel-core-2.23.2.jar:2.23.2]
[1] 	at org.apache.camel.processor.RedeliveryErrorHandler.process(RedeliveryErrorHandler.java:548) [org.apache.camel.camel-core-2.23.2.jar:2.23.2]
[1] 	at org.apache.camel.processor.CamelInternalProcessor.process(CamelInternalProcessor.java:201) [org.apache.camel.camel-core-2.23.2.jar:2.23.2]
[1] 	at org.apache.camel.processor.Pipeline.process(Pipeline.java:138) [org.apache.camel.camel-core-2.23.2.jar:2.23.2]
[1] 	at org.apache.camel.processor.Pipeline.process(Pipeline.java:101) [org.apache.camel.camel-core-2.23.2.jar:2.23.2]
[1] 	at org.apache.camel.processor.CamelInternalProcessor.process(CamelInternalProcessor.java:201) [org.apache.camel.camel-core-2.23.2.jar:2.23.2]
[1] 	at org.apache.camel.component.timer.TimerConsumer.sendTimerExchange(TimerConsumer.java:197) [org.apache.camel.camel-core-2.23.2.jar:2.23.2]
[1] 	at org.apache.camel.component.timer.TimerConsumer$1.run(TimerConsumer.java:79) [org.apache.camel.camel-core-2.23.2.jar:2.23.2]
[1] 	at java.util.TimerThread.mainLoop(Timer.java:555) [?:1.8.0_191]
[1] 	at java.util.TimerThread.run(Timer.java:505) [?:1.8.0_191]
[1] Caused by: org.apache.camel.ExpressionIllegalSyntaxException: **Illegal syntax: resource:platform:resources-data.txt**
[1] 	at org.apache.camel.support.LanguageSupport.loadResource(LanguageSupport.java:70) ~[org.apache.camel.camel-core-2.23.2.jar:2.23.2]
[1] 	at org.apache.camel.language.simple.SimpleLanguage.createExpression(SimpleLanguage.java:183) ~[org.apache.camel.camel-core-2.23.2.jar:2.23.2]
[1] 	at org.apache.camel.language.simple.SimpleLanguage.createExpression(SimpleLanguage.java:224) ~[org.apache.camel.camel-core-2.23.2.jar:2.23.2]
[1] 	at org.apache.camel.builder.SimpleBuilder.createExpression(SimpleBuilder.java:115) ~[org.apache.camel.camel-core-2.23.2.jar:2.23.2]
[1] 	... 11 more
[1] Caused by: java.io.FileNotFoundException: **Cannot find resource: platform:resources-data.txt in classpath for URI: platform:resources-data.txt**
[1] 	at org.apache.camel.util.ResourceHelper.resolveMandatoryResourceAsInputStream(ResourceHelper.java:196) ~[org.apache.camel.camel-core-2.23.2.jar:2.23.2]
[1] 	at org.apache.camel.support.LanguageSupport.loadResource(LanguageSupport.java:67) ~[org.apache.camel.camel-core-2.23.2.jar:2.23.2]
[1] 	at org.apache.camel.language.simple.SimpleLanguage.createExpression(SimpleLanguage.java:183) ~[org.apache.camel.camel-core-2.23.2.jar:2.23.2]
[1] 	at org.apache.camel.language.simple.SimpleLanguage.createExpression(SimpleLanguage.java:224) ~[org.apache.camel.camel-core-2.23.2.jar:2.23.2]
[1] 	at org.apache.camel.builder.SimpleBuilder.createExpression(SimpleBuilder.java:115) ~[org.apache.camel.camel-core-2.23.2.jar:2.23.2]
[1] 	... 11 more

--- a/examples/resources-route.groovy
+++ b/examples/resources-route.groovy
@@ -7,5 +7,5 @@
 from('timer:resources')
     .routeId('resources')
     .setBody()
-        .simple("resource:platform:resources-data.txt")
+        .simple("resource:classpath:resources-data.txt")
     .log('file content is: ${body}')

